l5kit/l5kit/environment/envs/l5_env.py

```diff
@@ -0,0 +1,333 @@
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, NamedTuple, Optional
+
+import gym
+import numpy as np
+import torch
+from gym import spaces
+from gym.utils import seeding
+
+from l5kit.configs import load_config_data
+from l5kit.data import ChunkedDataset, LocalDataManager
+from l5kit.dataset import EgoDataset
+from l5kit.environment.kinematic_model import KinematicModel, UnicycleModel
+from l5kit.environment.reward import L2DisplacementYawReward, Reward
+from l5kit.environment.utils import (calculate_kinematic_rescale_params, calculate_non_kinematic_rescale_params,
+                                     convert_to_numpy, KinematicActionRescaleParams, NonKinematicActionRescaleParams)
+from l5kit.rasterization import build_rasterizer
+from l5kit.simulation.dataset import SimulationConfig, SimulationDataset
+from l5kit.simulation.unroll import (ClosedLoopSimulator, ClosedLoopSimulatorModes, SimulationOutputCLE,
+                                     UnrollInputOutput)
+
+
+class SimulationConfigGym(SimulationConfig):
+    """Defines the default parameters used for the simulation of ego and agents around it in L5Kit Gym.
+    Note: num_simulation_steps should be eps_length + 1
+    This is because we (may) require to extract the initial speed of the vehicle for the kinematic model
+    The speed at start_frame_idx is always 0 (not indicative of the true current speed).
+    We therefore simulate the episode from (start_frame_idx + 1, start_frame_idx + eps_length + 1)
+    """
+    use_ego_gt: bool = False
+    use_agents_gt: bool = True
+    disable_new_agents: bool = False
+    distance_th_far: float = 30.0
+    distance_th_close: float = 15.0
+    num_simulation_steps: int = 33
+    start_frame_idx: int = 0
```

Comment:
Can you add docs here so Sphinx can capture and generate automatically ? The format here should be something like:

```
(...)
#: doc for the variable goes here
start_frame_idx: int = 0
```

Another alternative is to add it in the docstr above, feel free to check which one looks better after Sphinx renders it.